### ISSUE TITLE
Banning an already banned user failed message

### ISSUE THREAD

**Issue by cowballzach** [hicommonwealth/commonwealth#6171]
## Describe the bug
There is a failure message when trying to ban a user that has already been banned

## To Reproduce
Try to ban a user after they have already been banned. 

## Expected behavior
There should be some indicator that the user has already been banned. Should it say that the user is already banned?

## Additional context
<img width="419" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/144931524/f16a5be1-669b-40a3-b94c-759bc41e0aa6">

**Comment by CowMuon** [comment]
Mutation events (deprecated by W3C circa ~2011! IIANM) were finally removed in Chrome 127 (July 2023), replaced with MutationObserver, which presumably is causing this error (User is on Chrome >127.) It has been replaced with the MutationObserver API. 

At the very least we need to refactor any code where we are still mutating events to use the MutationObserver API. Expectation here is we won't just automatically convert, but will review code functions to determine whether we do need to be mutating in the first place; given the need for real time app updates to DOM, assumption is we will need to mutate, using MutationObservers. 

As such we should conduct a code audit for all such affected code where are are using a mutation event, and then -after confirming its desirable to mutate in the particular case- (vs. rewriting to not mutate) refactor the code.  Presumption is all refactoring can go in a single PR. 

All that being said, the particular case of banning an already banned user doesn't seem sufficient to scope this for Cycle 1. Unless it turns out there are nastier consequences of this deprecation waiting in the wings, default inclination is to kick the can on execution into Cycle 2 (Tho we will want to complete the research spike to confirm.) 

TLDR: 
- This was removed from Chrome in v.127. 
- We will need a quick product spike to understand/document full implications of refactoring. 
- Refactoring this needs to happen, tho unclear if needed for Denver @HIM92

**Comment by kurtisassad** [comment]
So this issue doesn't seem to be due to the Mutation Events. The issue here is that we classify all ban failures as "Ban Address Failed". The real fix here is to identify why the banAction failed and change the message in the toast accordingly.

The warning you see for Mutation Events being deprecated is being thrown from rocket-loader which is a tool from [Cloudflare](https://developers.cloudflare.com/speed/optimization/content/rocket-loader/). But it seems Cloudflare had updated this to remove this deprecation warning recently, as this log is no longer displayed in the console.

**Comment by CowMuon** [comment]
Really good investigation here Kurtis. Let's update the logic here to gracefully handle edge cases and inform the user what happened in a helpful toast.
